CHANGELOG-1.14.md

```diff
@@ -198,7 +198,9 @@ filename | sha512 hash
 * Fix a bug in server printer that could cause kube-apiserver to panic. ([#79349](https://github.com/kubernetes/kubernetes/pull/79349), [@roycaihw](https://github.com/roycaihw))
 * Fix a bug in the IPVS proxier where virtual servers are not cleaned up even though the corresponding Service object was deleted. ([#80942](https://github.com/kubernetes/kubernetes/pull/80942), [@gongguan](https://github.com/gongguan))
 
+## Known Issues
 
+* There is a known issue [#82923](https://github.com/kubernetes/kubernetes/issues/82923) in the AWS cloudprovider, where ELBs fail to register instances.  The fix has been merged into release-1.14 here [#82954](https://github.com/kubernetes/kubernetes/pull/82954).
```

Comment:
Thoughts on:

The fix has been merged ([#82954](https://github.com/kubernetes/kubernetes/pull/82954)) and will be included in v1.14.8.

Reply:
I was trying to stay away from future tense so that after it merges it doesn't sound weird but I think you're right, its better to explicitly say that.